packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1817,7 +1866,27 @@ function dispatchAction<S, A>(
 }
 
 function getCacheForType<T>(resourceType: () => T): T {
-  invariant(false, 'Not implemented.');
+  if (!enableCache) {
+    invariant(false, 'Not implemented.');
+  }
+  const cacheInstance: CacheInstance = readContext(CacheContext);
+  let cache = cacheInstance.cache;
+  if (cache === null) {
```

Comment:
This type is not allowed to be null. Is there some type cheatery we should fix or can we remove this branch?

Reply:
Yeah I changed the type when I added the root provider. Will remove.